data_tools/buildhub/jobs/scrape_archives.py

```diff
@@ -159,7 +151,7 @@ def archive_url(product, version=None, platform=None, locale=None, nightly=None)
 async def fetch_nightlies(session, queue, product, client):
     # Check latest known version on server.
     filters = {
-        "source.product": product,
+        "target.product": product,
```

Comment:
Because after multiple discussion we said that it didn't made sense to have it separated from target.